src/test/java/integration/EventsTest.java

```diff
@@ -882,8 +884,9 @@ public void given_multibranchWithSourcesAndTwoBranches_when_indexing_then_deadBr
             prj.scheduleBuild2(0).getFuture().get();
             r.waitUntilNoActivity();
             assertThat("Indexing applies dead branch cleanup",
-                    prj.getItems(), containsInAnyOrder(master));
```

Comment:
The reason for using `containsInAnyOrder(_some_single_item_)` rather than `is(Collections.singleton(_some_single_item_))` is that you hit a lot of trouble trying to mash the generic collection types together, whereas `containsInAnyOrder(_some_single_item_)` doesn't care what type of collection you have been given